Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/RelationshipsStepDefinitions.cs

```diff
@@ -207,4 +256,13 @@ private async Task<ApiResponse<RelationshipMetadata>> CreateRelationship(Client
 
         return await client.Relationships.CreateRelationship(createRelationshipRequest);
     }
+
+    private async Task<ApiResponse<RelationshipMetadata>> RejectRelationship(Client client, string relationshipId)
+    {
+        var rejectRelationshipRequest = new RejectRelationshipRequest
+        {
+            CreationResponseContent = "AAA".GetBytes()
+        };
+        return await client.Relationships.RejectRelationship(relationshipId, rejectRelationshipRequest);
+    }
```

Comment:
Is this even used?

Reply:
No, forgot to remove it after using the new helper method `CreateRejectedRelationshipBetween`